### PR TITLE
README: move to Markdown format and fix example

### PR DESCRIPTION
**Pull request by yegorich**
Fix print statements to be compatible with Python3.

Rename README.txt to README.md.

Some PEP8 fixes.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>

### CODE REVIEW

**Comment by yegorich** [comment]
I've pushed v2 that converts README to rst format, so that it could be used in PyPI.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Pyiface is a package that exposes the network interfaces of the operating
+system in a easy to use and transparent way Typical usage looks like this:
+
+.. code-block:: python
+
+    #!/usr/bin/env python
+    
+    import pyiface
+    from pyiface.ifreqioctls import IFF_UP
```
Comment (alexvasiu) [inline README.rst]: * `pyiface.ifreqioctls` it's not visible
* also socket should be imported

```python3
import socket
IFF_UP = 0x1
```
I think, we should make `pyiface.ifreqioctls` visible. I can do that.

**Comment by yegorich** [comment]
What are we going to do with this PR? Merge and then @alexvasiu would send a follow up patch?

**Comment by alexvasiu** [comment]
@yegorich I think that's the way we should do. Merge this and I'll submit my patch (I hope this week) and then I'll release a new version in **pip** with these changes.

**Comment by yegorich** [comment]
FYI: PyIface is now on [OpenHUB](https://www.openhub.net/p/PyIface)

**Comment by yegorich** [comment]
@bat-serjo ping

File: README.rst (hunk 1 above)
Comment (alexvasiu) [reply README.rst]: Actually, it works now :+1:

Comment (yegorich) [reply README.rst]: @alexvasiu so release can be made now?

Comment (alexvasiu) [reply README.rst]: Sure :) I'll make a release now in **pip** and then I'll also update recipe in `meta-openembedded`

**Pull request merged**